packages/cyberstorm/src/components/Toast/Toast.module.css

```diff
@@ -0,0 +1,181 @@
+.root {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 12px 24px 5px rgb(0 0 0 / 0.65);
+}
+
+.contentWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-left: 3px solid var(--feature-color);
+  border-radius: var(--border-radius--4);
+  background-color: var(--background-color);
+
+  --background-color: var(--alert-info-bg-color);
```

Comment:
Does the naming `--alert-` cause possible problems, i.e. someone editing the values assuming it affects only the alert component? Dunno what would be a better name, just pointing out that I think it's usually frowned upon to bind the names to specific components and then reusing them in other components.